ui/main_page.go

```diff
@@ -133,11 +141,7 @@ func (mp *mainPage) OnResume() {
 
 	mp.updateBalance()
 
-	if pg, ok := mp.pages[mp.current]; ok {
-		pg.OnResume()
-	} else {
-		mp.changeFragment(OverviewPage(mp.pageCommon), PageOverview)
-	}
+	mp.changeFragment(page.NewOverviewPage(mp.load), page.Overview)
```

Comment:
Check for the current page before switching to overview.
```suggestion
	if pg, ok := mp.pages[mp.current]; ok {
		pg.OnResume()
	} else {
		mp.changeFragment(OverviewPage(mp.pageCommon), PageOverview)
	}
```